packages/patrol_cli/lib/src/features/test/ios_deploy.dart

```diff
@@ -0,0 +1,77 @@
+import 'dart:io';
+
+import 'package:dispose_scope/dispose_scope.dart';
+import 'package:file/file.dart';
+import 'package:patrol_cli/src/common/extensions/process.dart';
+import 'package:patrol_cli/src/common/logger.dart';
+import 'package:process/process.dart';
+
+class IOSDeploy {
+  IOSDeploy({
+    required ProcessManager processManager,
+    required FileSystem fs,
+    required DisposeScope parentDisposeScope,
+    required Logger logger,
+  })  : _processManager = processManager,
+        _fs = fs,
+        _disposeScope = DisposeScope(),
+        _logger = logger {
+    _disposeScope.disposedBy(parentDisposeScope);
+  }
+
+  final ProcessManager _processManager;
+  final FileSystem _fs;
+  final DisposeScope _disposeScope;
+  final Logger _logger;
+
+  /// Installs and launches the app on physical iOS device with [deviceId].
+  ///
+  /// To kill the app, send `kill` to the returned [Process]' stdin.
```

Comment:
```suggestion
  /// To kill the app, send `kill` to the returned [Process]'s stdin.
```

Use `'s` for singular possessors, even if they end with -s

Reply:
right